orc8r/cloud/go/obsidian/swagger/handlers/handlers.go

```diff
@@ -14,17 +14,84 @@
 package handlers
 
 import (
+	"bytes"
+	"html/template"
 	"net/http"
+	"strings"
 
 	"magma/orc8r/cloud/go/obsidian"
 	"magma/orc8r/cloud/go/obsidian/swagger"
+	"magma/orc8r/cloud/go/orc8r"
+	"magma/orc8r/lib/go/registry"
 
 	"github.com/labstack/echo"
+	"github.com/labstack/echo/middleware"
+	"github.com/pkg/errors"
 )
 
-// GetGenerateCombinedSpecHandler returns a routing handler which creates
+// Defines the struct to be templated into Swagger UI.
+type UIInfo struct {
+	// Defines the URL to be injected into the Swagger UI template
+	URL string
+}
+
+// RegisterSwaggerHandlers registers routes for Swagger specs and
+// the Swagger UI.
+func RegisterSwaggerHandlers(e *echo.Echo) error {
+	trailSlashMiddleware := middleware.AddTrailingSlashWithConfig(middleware.TrailingSlashConfig{
+		RedirectCode: http.StatusMovedPermanently,
```

Comment:
Is this the default? I.e. do you have to use `WithConfig` ?

Reply:
Yup, we have to use add this version of TrailingSlash(withConfig) because otherwise it won't redirect.